src/components/Checklist.js

```diff
@@ -1,48 +1,73 @@
 import React from 'react'
+import { ChecklistHeader,
+         ChecklistTitle,
+         ChecklistToolbar,
+         ChecklistItems,
+         ChecklistAddItemForm,
+         ChecklistFooter } from './'
+import { addArrayItem, generateId } from '../helpers/helpers'
 
 export class Checklist extends React.Component {
+
+  constructor () {
+    super()
```

Comment:
Why should we add props argument to constructor? This component doesn't required props at all.